app/components/trainees/filters/view.html.erb

```diff
@@ -37,47 +36,38 @@
         </div>
       <% end %>
 
-        <div class="moj-filter__options">
+      <div class="moj-filter__options">
 
-          <form method="get">
-            <%= submit_tag "Apply filters", class: "govuk-button" %>
+        <form method="get">
```

Comment:
Just wondering could we used the form builder?

Reply:
I don't think we can - I tried to use the form builder when originally building the filters but couldn't then explicitly set the `checked` or `selected` values on the fields based on the active params.